packages/roosterjs-editor-plugins/lib/plugins/Announce/AnnounceHandlerImpl.ts

```diff
@@ -0,0 +1,69 @@
+import type { AnnounceData, DefaultAnnounceStrings } from 'roosterjs-editor-types';
+
+/**
+ * @internal
+ */
+export default class AnnounceHandler {
```

Comment:
Do we need to have this class? Can this be merged into the plugin?

Reply:
Merged both classes as suggested